hack/update-api-reference-docs.sh

```diff
@@ -71,6 +71,7 @@ for ver in $VERSIONS; do
   docker run ${user_flags} \
     --rm -v "${TMP_IN_HOST}":/output:z \
     -v "${SWAGGER_PATH}":/swagger-source:z \
+    --net=host -e "https_proxy=${KUBERNETES_HTTPS_PROXY}" \
```

Comment:
this probably needs to default to empty if not set, i.e. `${KUBERNETES_HTTPS_PROXY:-}`

alternately, only add the flag when the env var is set:
```bash
${KUBERNETES_HTTPS_PROXY:+"-e https_proxy='${KUBERNETES_HTTPS_PROXY}'"}
```